src/main/java/com/facebook/openwifirrm/ucentral/UCentralClient.java

```diff
@@ -142,6 +142,17 @@ public static void verifySsl(boolean enable) {
 	 */
 	private WebTokenResult accessToken;
 
+	/**
+	 * The unix timestamp (in seconds) keeps track of when the accessToken is created.
+	 */
+	private long created;
+
+	/**
+	 * The unix timestamp (in seconds) keeps track of last time when the accessToken 
+	 * is accessed.
+	 */
+	private long lastAccess;
+
```

Comment:
can you pick better names like "accessTokenCreated" and "accessTokenLastAccess"?